tests/models/jukebox/test_modeling_jukebox.py

```diff
@@ -363,7 +369,8 @@ def test_sampling(self):
         self.assertIn(zs[2][0].detach().cpu().tolist(), [self.EXPECTED_OUTPUT_0, self.EXPECTED_OUTPUT_0_PT_2])
 
     @slow
-    @require_torch_gpu
+    @require_torch_accelerator
+    @require_torch_fp16
```

Comment:
why fp16 here?

Reply:
Oops, the fp16 decorator was misused here. Thanks for pointing it out